### ISSUE TITLE
off-the-grid MARL environment is not allowed to download! 

### ISSUE THREAD

**Issue by Guowei-Zou** [zbzhu99/madiff#5]
Mr Zhu,
i'm sorry to bother you that i found that the ural of MAMuJoCo is not allowed to download any more. (https://sites.google.com/view/og-marl).
Are you kind enough to help me to solve this problem. I will appropriate you so much!

Guowei Zou.
Sun Yet-San University(Guangzhou)

**Comment by Guowei-Zou** [comment]
This XML file does not appear to have any style information associated with it. The document tree is shown below.
<Error>
<Code>NoSuchBucket</Code>
<BucketName>offline-marl-dataset</BucketName>
<RequestId>tx00000d1ba668182594d8c-0066ece8ab-10413f21-instadeep</RequestId>
<HostId>10413f21-instadeep-instadeep</HostId>
</Error>

**Comment by zbzhu99** [comment]
Hi @Guowei-Zou, it seems the maintainers of og-marl have replaced their original download links with vault-based datasets that require [specific APIs](https://colab.research.google.com/github/instadeepai/og-marl/blob/main/examples/dataset_api_demo.ipynb) to access. To make things easier, I've updated all the dataset links in the README with OneDrive links I created, which still point to the original datasets. I may explore using their new API in the future, depending on availability.

Best regards,
Zhengbang
